tiatoolbox/tools/stainnorm.py

```diff
@@ -33,7 +33,7 @@
 )
 from tiatoolbox.utils.exceptions import MethodNotSupported
 from tiatoolbox.utils.misc import load_stain_matrix
-from tiatoolbox.utils.transforms import convert_OD2RGB, convert_RGB2OD
+from tiatoolbox.utils.transforms import convert_od2rgb, convert_rgb2od
```

Comment:
Could simply call them `od2rgb` and `rgb2od` in line with skimage colorspace conversions.

Reply:
Resolved in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/263/commits/5fcf23f673d8dbc274080367148591fc20d40d34